gtsfm/frontend/verifier/colmap_two_view_geometry_verifier.py

```diff
@@ -0,0 +1,195 @@
+"""
```

Comment:
not sure if this file is named appropriately, technically loransac is also being used from colmap. so why not call this gric_verifer?

Reply:
You're right. I'll rename to `GricVerifier` and `gric_verifier.py`